database/mysql/create_testing_table.sql

```diff
@@ -49,6 +49,7 @@ CREATE OR REPLACE TABLE `item_testing` (
 `evaltype` INT(11) NOT NULL DEFAULT '0',
 `jmx_endpoint` VARCHAR(255) NOT NULL DEFAULT '' COLLATE 'utf8_bin',
 `master_itemid` BIGINT(20) UNSIGNED NULL DEFAULT NULL,
+`timeout` VARCHAR(255) NOT NULL DEFAULT '3s' COLLATE 'utf8mb4_bin',
```

Comment:
Should all VARCHAR columns have same utf8mb4_bin collation?

Reply:
I think collation was something that was part of suggested 6.0 migrations. Here's the document: https://www.zabbix.com/documentation/current/en/manual/appendix/install/db_charset_coll

Not sure if this is now listed in migration notes but it should be IMO. @jahietan

Reply:
fixed collation to match others

Reply:
Not sure if that's the right direction since Zabbix recommends converting to utf8mb4_bin which requires doin migration linked above. So to my understanding in properly setup Zabbix 6.0 all VARCHAR columns should use utf8mb4_bin collation and thus so should item_testing table.

Reply:
Just checked that our current version of MariaDB doesn't support utf8mb4 at all so migration to that isn't possible and this is correct.